src/api/routes/routes.admin.js

```diff
@@ -14,4 +14,10 @@ router.post(
   AdminController.registerBus
 );
 
+router.post(
+  '/create-trip',
+  Model(Schema.createTrip, 'payload'),
+  AdminController.createTrip
```

Comment:
is this not a protected route?... only an admin should have access to it.. the authorization middleware is not there

Reply:
The "authenticate" middleware is at the top, above other routes.